.claude/skills/bitwarden-reviewing-changes/SKILL.md

```diff
@@ -0,0 +1,65 @@
+---
+name: bitwarden-reviewing-changes
+description: Claude code review standards for Bitwarden repositories.
+---
```

Comment:
Front-matter should contain
- Version
- Tool permissions

While both are technically optional per the spec, we should treat them as required. Version for obvious reasons; Tool permissions should be explicitly listed so that the Skill isn't inadvertently granted permissions it should not have. Being focused on _reviewing_ code, we should limit this to readonly tools except those required to make PR comments.